sites/main-site/src/content/blog/2025/maintainers-minutes-2025-02-28.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "Maintainers Minutes: February 2025"
+subtitle: "Keeping you informed of the latest maintainers discussions"
+pubDate: 2025-02-28T18:00:00+01:00
+headerImage: "/assets/images/blog/maintainers-minutes-2024-07-26/maintainers-wide.png"
+headerImageAlt: "Cartoon yellow rubber duck with nf-core logo badge on its body with the nf-core logo."
+embedHeaderImage: true
+authors:
+    - "jfy133"
+label:
+    - "maintainers"
+---
+
+import Profile from "@components/GitHubProfilePictureExtended.svelte";
+import { Image } from "astro:assets";
+
+The 'Maintainers Minutes' aims to give further insight into the workings of the [nf-core maintainers team](/governance#maintainers)
+by providing brief summaries of the monthly team meetings.
+
+## Overview
+
+In the January meeting we discussed (amongst others) the following topics:
+
+- [Spring cleaning](#spring-cleaning)
+- [Merchandise](#merchandise)
+- [Hackathon projects](#hackathon)
+- [Smooth running!?](#smooth-running)
+
+## Spring cleaning
+
+This month we again discussed the upcoming pre-hackathon spring cleaning across all common nf-core modules (and more!).
+
+As announced last month this will be happening between March 10th and March 14th, where we will be deleting merged branches, archiving abandoned pipelines, and general papercuts to keep everything spick-and-span.
```

Comment:
```suggestion
As announced last month this will be happening between March 10th and March 14th, where we will be deleting merged branches, archiving abandoned pipelines, and general [papercuts bugs](https://en.wikipedia.org/wiki/Paper_cut_bug) to keep everything spick-and-span.
```